Weather Man/file_parser.py

```diff
@@ -0,0 +1,73 @@
+# weather_parser.py
+
+import os
+import re
+from datetime import datetime
+from weather_data import WeatherData
+import pandas as pd
+
+
+def parse_file(file_path):
```

Comment:
Why is this function out of class? Stand-alone method should not be used until neccessary